templates/strapi/files/backend/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Create quickstart Strapi project.
+yarn create strapi-app tmp-app --quickstart --no-run
```

Comment:
Hrm.  So... I'm torn here.

On the one hand, doing a project create in the build hook and copying files around seems like it's going to result in a project that works for first install and not much else.  Certainly it would be hard to run locally, wouldn't it?  

On the other, the alternative seems to be to commit a whole lot of code to the repo.  Which... to be fair, is what we're doing for most other projects that need a "create-project" step.  (Mostly PHP apps.)

I'm not sure which approach is less bad.

Reply:
> On the one hand, doing a project create in the build hook and copying files around seems like it's going to result in a project that works for first install and not much else. Certainly it would be hard to run locally, wouldn't it?

I think it will still work fine regardless of first install, but yet running locally would require you run `platform build` or an equivalent npm script that does this.

> On the other, the alternative seems to be to commit a whole lot of code to the repo. Which... to be fair, is what we're doing for most other projects that need a "create-project" step. (Mostly PHP apps.)

I can always go back to the way I had it before if that makes more sense. In that case I had had updated the areas where the upstream `localhost` linking issue was there.